### DISCUSSION TITLE
Lock option from removal?

### DISCUSSION THREAD

**Discussion by seabasss** [orchidjs/tom-select#432] [General]
I know I can lock and disable a select box and disable an option from being added, but can I pre-populate options and lock them from being removed from the selection?

E.g. I want certain options to always be added and not able to be removed and other options can be added and removed freely.

Thanks!

**Comment by oyejorge** [comment]
Could probably do this with https://tom-select.js.org/docs/#ondeletevalues-event

**Reply by seabasss** [reply]
Looks good, thanks!

**Reply by seabasss** [reply]
Think I need some more guidance here. Any way to have `onDelete(values, event)` include the data or selected items? I have the values, but how do I use that to check if a removed selected item has `data-disabled` added to it so I can return false? Thank you!
